api/rocdecode.h

```diff
@@ -782,6 +782,599 @@ typedef struct _RocdecHevcIQMatrix {
     uint32_t                reserved[4];
 } RocdecHevcIQMatrix;
 
+/** \brief Segmentation Information for AV1
+  */
+typedef struct _RocdecSegmentationStructAV1 {
+    union {
+        struct {
+            /** Indicates whether segmentation map related syntax elements
+            *  are present or not for current frame. If equal to 0,
+            *  the segmentation map related syntax elements are
+            *  not present for the current frame and the control flags of
+            *  segmentation map related tables feature_data[][], and
+            *  feature_mask[] are not valid and shall be ignored by accelerator.
+            */
+            uint32_t         enabled                                     : 1;
```

Comment:
Please remove extra spaces for this file's struct members, including the spaces between datatypes and variable names, spaces between variable names and equal signs, etc.
For example change `uint32_t               enabled                        : 1;` to 
`uint32_t enabled : 1;`